tcp_queue_length/README.md

```diff
@@ -0,0 +1,79 @@
+# Agent Check: TCP Queue Length
+
+## Overview
+
+This check monitors the size of the Linux TCP receive and send queues.
+
+## Setup
+
+### Installation
+
+`tcp_queue_length` is a core Agent 6/7 check that relies on an eBPF part implemented in `system-probe`.
+
+The eBPF program used by `system-probe` is compiled at runtime and requires you to have access to the proper kernel headers.
+
+On Debian-like distributions, install the kernel headers like this:
+```sh
+apt install -y linux-headers-$(uname -r)
+```
+
+On RHEL-like distributions, install the kernel headers like this:
+```sh
+yum install -y kernel-headers-$(uname -r)
+```
+
+### Configuration
+
+Enabling the `tcp_queue_length` integration requires both the `system-probe` and the core agent to have the configuration option enabled.
+
+Inside the `system-probe.yaml` configuration file, the following parameters must be set:
+```yaml
+system_probe_config:
+  enabled: true
+  enable_tcp_queue_length: true
+```
+
+1. Edit the `tcp_queue_length.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your tcp_queue_length performance data.
+   See the [sample tcp_queue_length.d/conf.yaml][1] for all available configuration options.
+
+2. [Restart the Agent][3].
+
+The `only_count_nb_contexts` parameter controls whether:
+* the check should collect all the data—that is, the size of the TCP queues for every single connection (`false`)
+* the check should collect only the number of timeseries it would normally collect (`true`). This is the default.
```

Comment:
```suggestion
1. Edit the `tcp_queue_length.d/conf.yaml` file, in the `conf.d/` folder at the root of your
   Agent's configuration directory to start collecting your tcp_queue_length performance data.
   See the [sample tcp_queue_length.d/conf.yaml][1] for all available configuration options.

    The `only_count_nb_contexts` parameter is enabled by default and collects only the number of timeseries it would normally collect. If disabled, the check will collect all data-- that is, the size of the TCP queues for every single connection.

2. [Restart the Agent][3].

```

I think formatting the option like this might be easier to read and understand it's referencing the configuration file.

Reply:
Sorry the diff looks like a lot changed, but I just reworded the section about `only_count_nd_contexts` and moved it after Step 1.